models/message.js

```diff
@@ -1,38 +1,95 @@
-Chatter.Message = ChatterMessage = Astro.Class({
-    name: "ChatterMessage",
-    collection: new Mongo.Collection("chattermessage"),
+const reqStrNotNull = Validators.and([
+  Validators.required(),
+  Validators.string(),
+  Validators.notNull()
+]);
 
-    fields: {
-        userId: {
-            type: "string"
-        },
+Chatter.Message = ChatterMessage = Astro.Class({
+  name: "ChatterMessage",
+  collection: new Mongo.Collection("chattermessage"),
 
-        roomId: {
-            type: "string"
-        },
+  fields: {
+    userId: {
+      type: "string"
+    },
 
-        userNick: {
-            type: "string"
-        },
+    roomId: {
+      type: "string"
+    },
 
-        userAvatar: {
-            type: "string"
-        },
+    message: {
+      type: "string"
+    },
 
-        message: {
-            type: "string"
-        }
+    nickname: {
+      type: "string"
     },
 
-    events: {
+    avatar: {
+      type: "string"
+    }
+  },
 
-    },
+  validators: {
```

Comment:
Easier to have validators in their respective field definitions instead of separately? If so, it can wait until later, not a big deal.